src/gpig/rrs_inversion_pigments.py

```diff
@@ -464,7 +464,7 @@ def betasw124_ZHH2009(lambda_, S, Tc, delta=0.039):
 
 def tempsal_corr(lambda_):
     """ """
-    with resources.path(__name__, "resources/Sullivan_pure_water_temp_sal.csv") as p:
+    with resources.path("gpig.resources", "Sullivan_pure_water_temp_sal.csv") as p:
         pwts = pd.read_csv(p)
```

Comment:
```suggestion
    with resources.path(__package__, "resources", "Sullivan_pure_water_temp_sal.csv") as p:
        pwts = pd.read_csv(p)
```

Can you try if this works? We want to avoid hard coding the package name where possible.

Reply:
Darn, I was reading python 3.13 docs. As is works.